doc/rados/operations/cache-tiering.rst

```diff
@@ -62,6 +62,28 @@ configure how this migration takes place. There are two main scenarios:
   allows the workload to function properly while the cache is drained,
   without adding any new objects to the cache.
 
+Other cache modes are:
+
+- **Read-only** promotes objects to the cache on read operations only; write
+  operations are forwarded to the base tier. When objects are updated in the
+  base tier, Ceph makes no attempt to sync these updates to the corresponding
+  objects in the cache. This mode is intended for read-only workloads that do
+  not require consistency to be enforced by the storage system.
+
+- **Read-forward** does not promote objects from the base tier on read.
+  In other words, read requests that miss in the cache are forwarded to the
+  base tier. Write operations on the cache are accepted and flushed to the base
+  tier. This mode is useful when the base and cache tiers are comparably fast
+  for reads, but there is asymmetric performance or endurance for writes.
```

Comment:
I think we should not mention the 'forward' modes, and (ideally) we should remove them entirely.  They do not work right.  We can make exsiting cache tiers using it switch to read-proxy on upgrade.